### PR TITLE
JBIDE-20242: Forge New and Noteworthy for 4.3.0.Beta2

### PR DESCRIPTION
**Pull request by koentsje**

### CODE REVIEW

**Comment by maxandersen** [comment]
@rultor hello

**Comment by maxandersen** [comment]
@rultor please check and merge this pull request

**Comment by maxandersen** [comment]
@rultor merge

**Comment by maxandersen** [comment]
@rultor merge again please, enabled root this time.

**Comment by maxandersen** [comment]
@rultor try merge again

**Comment by maxandersen** [comment]
@rultor merge ...crossing fingers ;)

**Comment by maxandersen** [comment]
merged in manually - sorry for the rultor noise ;)

File: documentation/whatsnew/forge/forge-news-4.3.0.Beta2.adoc [hunk 1]
```diff
@@ -0,0 +1,15 @@
+= Forge Tools 4.2.0.Beta2 What's New
+:page-layout: whatsnew
+:page-component_id: forge
+:page-component_version: 4.2.0.Beta2
```
Comment (alexeykazakov) [inline documentation/whatsnew/forge/forge-news-4.3.0.Beta2.adoc]: Should be 4.3.0... Not 4.2.0...

File: documentation/whatsnew/forge/forge-news-4.3.0.Beta2.adoc [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+:page-product_id: jbt_core 
+:page-product_version: 4.2.0.Beta2
```
Comment (alexeykazakov) [inline documentation/whatsnew/forge/forge-news-4.3.0.Beta2.adoc]: The same typo.

**Pull request closed without merging**